Assignment/solution_student/index.html

```diff
@@ -16,15 +18,62 @@
     <div class="container">
     
       <header>
-        <h1>FEWD HTML Template</h1>
+        <h1>Relaxr</h1>
+        <ul>
+          <li>About</li>
+          <li>FAQ</li>
+          <li>Team</li>
+          <li>Contact Us</li>
+          <li>Blog</li>
+        </ul>
       </header>
 
       <main>
-        <p>Our HTML starter template!</p>
+        <div class="floatleft">
+        <article>
```

Comment:
You were spot on when choosing an `article` tag to markup the blog post, but you included both posts in a single `article` tag, but they are two distinct articles, so I think it would make the most sense to keep them separated in their own `article` tag :)